regression/scripts/variable_cruise_distance/variable_cruise_distance.py

```diff
@@ -40,9 +38,14 @@ def main():
     
     plot_results(results)
     
-    error = abs(mission.target_landing_weight - results.conditions.weights.total_mass[-1,0])
-    print('landing weight error' , error)
-    assert error < 1.
+    distance_regression = 4317710.33719722
+    distance_calc       = results.conditions.frames.inertial.position_vector[-1,0]
+    error_distance      = abs((distance_regression - distance_calc )/distance_regression)
+    assert error_distance < 1e-6
+    
+    error_weight = abs(mission.target_landing_weight - results.conditions.weights.total_mass[-1,0])
+    print('landing weight error' , error_weight)
+    assert error_weight < 1.
```

Comment:
Add a comment with the reason for < 1 instead of 1e-6

Reply:
I kept this part as was in the original. Now's its 10^-6